modules/ztp-setting-managed-bare-metal-host-kernel-arguments-manual-workflow.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+[id="setting-managed-bare-metal-host-kernel-arguments_{context}"]
+= Setting managed bare-metal host kernel arguments
```

Comment:
Again, I would like to see this title be more descriptive of the actual task to differentiate it in Google searches, etc.